h2o-core/src/main/java/water/parser/ParseTime.java

```diff
@@ -45,20 +45,40 @@ public static boolean isTime(BufferedString str) {
   };
 
   public static long attemptTimeParse( BufferedString str ) {
+    List<Throwable> exceptions = new ArrayList<>();
     try {
       long t0 = attemptYearFirstTimeParse(str); // "yyyy-MM-dd" and time if present
-      if( t0 != Long.MIN_VALUE ) return t0;
+      if (t0 != Long.MIN_VALUE) return t0;
+    } catch (IllegalFieldValueException | IllegalInstantException | ArrayIndexOutOfBoundsException e) {
+      exceptions.add(e);
+    }
+
+    try {
       long t1 = attemptDayFirstTimeParse1(str); // "dd-MMM-yy" and time if present
-      if( t1 != Long.MIN_VALUE ) return t1;
+      if (t1 != Long.MIN_VALUE) return t1;
+    } catch (IllegalFieldValueException | IllegalInstantException | ArrayIndexOutOfBoundsException e) {
+      exceptions.add(e);
+    }
+    try {
       long t2 = attemptYearMonthTimeParse(str); // "yy-MMM", not ambiguous with dd-MMM-yy because of trailing "-yy"
-      if( t2 != Long.MIN_VALUE ) return t2;
+      if (t2 != Long.MIN_VALUE) return t2;
+    } catch (IllegalFieldValueException | IllegalInstantException | ArrayIndexOutOfBoundsException e) {
+      exceptions.add(e);
+    }
+    try {
       long t3 = attemptTimeOnlyParse(str); // Time if present, no date
-      if( t3 != Long.MIN_VALUE ) return t3;
+      if (t3 != Long.MIN_VALUE) return t3;
+    } catch (IllegalFieldValueException | IllegalInstantException | ArrayIndexOutOfBoundsException e) {
+      exceptions.add(e);
+    }
+    try {
       long t4 = attemptDayFirstTimeParse2(str); // "dd/MM/yy" and time if present; note that this format is ambiguous
-      if( t4 != Long.MIN_VALUE ) return t4;     // Cant tell which date: 3/2/10 is
-    } catch( org.joda.time.IllegalFieldValueException | // Not time at all
-             org.joda.time.IllegalInstantException      // Parsed as time, but falls into e.g. a daylight-savings hour hole
-             ie ) { } //FIXME should collect errors and report at end of parse
+      if (t4 != Long.MIN_VALUE) return t4;     // Cant tell which date: 3/2/10 is
+    } catch (IllegalFieldValueException | IllegalInstantException | ArrayIndexOutOfBoundsException e) {
+      exceptions.add(e);
+    }
+
+    Log.err(exceptions);
```

Comment:
This will pollute the logs, any string column is being tested for being a date-time column?